tests/io/aims/sets/test_static_restart_from_relax_generator.py

```diff
@@ -2,14 +2,13 @@
 
 from __future__ import annotations
 
-from pathlib import Path
-
 from pymatgen.io.aims.sets.core import StaticSetGenerator
-from pymatgen.util.testing.aims import O2, Si, comp_system
+from pymatgen.util.testing import TEST_FILES_DIR
+
+from ..conftest import O2, Si, comp_system  # noqa: TID252
```

Comment:
```suggestion
from tests.conftest import O2, Si, comp_system
```

Reply:
Can you double check this? This doesn't seem to work (we have to import from `pymatgen/tests/io/aims/conftest.py` not `pymatgen/tests/conftest.py`). Also `pymatgen.tests` isn't made a module at this moment (not sure if absolute import would work).

```python
____________________________________ ERROR collecting tests/io/aims/sets/test_bs_generator.py ____________________________________
ImportError while importing test module '/Users/yang/developer/pymatgen/tests/io/aims/sets/test_bs_generator.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/io/aims/sets/test_bs_generator.py:8: in <module>
    from tests.conftest import Si, comp_system  # noqa: TID252
E   ImportError: cannot import name 'Si' from 'tests.conftest' (/Users/yang/developer/pymatgen/tests/conftest.py)
```

Reply:
ah, then it should work with `from tests.io.aims.conftest import ...`

Reply:
It doesn't seem to work on my machine either. I thought we could only:
1. Relative import, which is used right now but the `ruff` rule TID252 would suggest against this.
2. Absolute import, with `pymatgen.tests.io.aims.conftest`, but this would not work because `pymatgen.tests` are not a package in our case (no `__init__.py`).

Perhaps just ignore this rule for tests?
> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:

Reply:
> Perhaps just ignore this rule for tests?

sounds good to me. the absolute import from[ `tests` works in `pymatviz`](https://github.com/janosh/pymatviz/blob/1de350a146b53ae03f343ff9a78454ba7c976186/tests/test_bar.py#L12) but that's because it has an `__init__.py`

Reply:
ok!